### PR TITLE
Touchscreen support with pygame 2.0.0

### PR DESCRIPTION
**Pull request by werdeil**
Hello everyone,

As discussed in #222, `pygame` now handles the touchscreen separately from mouse.

Here is a first draft of the integration, I'm not sure of everything and I'll be happy to correct anything if you want.

I've tested it on a raspberry pi with a 7" touchscreen and it works. I have some issues with the back button (doesn't always work but maybe it is because it is quite small on the touchscreen.

### CODE REVIEW

**Comment by ppizarror** [comment]
Excellent! Added minimum details, but I can't test it; anyway, it does not break any of the current code.
I would suggest, only for development purposes, to set touchscreen_enable=False, and only set it to True if we're 100% certain that this will work in any scenario.

File: pygame_menu/widgets/widget/textinput.py [hunk 1]
```diff
@@ -909,6 +917,22 @@ def _check_mouse_collide_input(self, pos):
             topleft, _ = self._rect.topleft
             self._update_cursor_mouse(mousex - topleft)
             return True  # Prevents double click
+        self._cursor_render = True
```
Comment (ppizarror) [inline pygame_menu/widgets/widget/textinput.py:920]: Why this? And if helps to fix some issues, why check_touch_collide_input does not force to render again the cursor?

**Comment by ppizarror** [comment]
Hi @anxuae! are you able to test this feature in a touchscreen environment?

File: pygame_menu/widgets/widget/textinput.py (hunk 1 above)
Comment (werdeil) [reply pygame_menu/widgets/widget/textinput.py:920]: It is clearly an error that I may have added when I was trying to see if in needed to copy the _update_cursor_mouse for the touchscreen or use it directly, I'll push a correction.

**Comment by werdeil** [comment]
I know that @anxuae is able to test this feature, we can clearly wait for his tests :-)

**Comment by ppizarror** [comment]
I'll merge by now. Touchscreen will be disabled by default
thanks!

**Pull request merged**